WORKSPACE

```diff
@@ -72,42 +72,42 @@ go_download_sdk(
     name = "go_sdk_linux",
     goarch = "amd64",
     goos = "linux",
-    version = "1.21.3",  # Keep in sync with .github/workflows/checkstyle.yaml
+    version = "1.21.4",  # Keep in sync with .github/workflows/checkstyle.yaml
```

Comment:
should we use a variable for the SDK version?

Reply:
Good idea. Should make the diff smaller.

Pushed a change for it.